pom.xml

```diff
@@ -433,6 +434,20 @@
                 <version>${icu.version}</version>
             </dependency>
 
+            <!-- We need to fix our Scala versions as otherwise our depdendencies bring in
```

Comment:
nit: can we update the comment to make it more clear that this is a temporary workaround?